<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>texture_widget</class>
 <widget class="QWidget" name="texture_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="m_type_lbl">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_tex_type_cmb">
       <item>
        <property name="text">
         <string>2D Texture</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cubemap</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_image_file_lbl">
       <property name="text">
        <string>Image File</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLineEdit" name="m_image_file_le"/>
       </item>
       <item>
        <widget class="QPushButton" name="m_image_file_tb">
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_cm_gb">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Cubemap Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Front</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="m_cm_front_le"/>
          </item>
          <item>
           <widget class="QPushButton" name="m_cm_front_tb">
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="m_cm_back_le"/>
          </item>
          <item>
           <widget class="QPushButton" name="m_cm_back_tb">
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="m_cm_right_le"/>
          </item>
          <item>
           <widget class="QPushButton" name="m_cm_right_tb">
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="m_cm_left_le"/>
          </item>
          <item>
           <widget class="QPushButton" name="m_cm_left_tb">
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Top</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="m_cm_top_le"/>
          </item>
          <item>
           <widget class="QPushButton" name="m_cm_top_tb">
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Bottom</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="m_cm_bottom_le"/>
          </item>
          <item>
           <widget class="QPushButton" name="m_cm_bottom_tb">
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.642045 rgba(30, 30, 30, 255), stop:1 rgba(48, 48, 48, 255));
border-radius: 3px;
border: 1px solid rgb(100,100,100);
padding: 5px;
color: white;
min-width: 15px;
}

QPushButton:disabled {
background-color: rgb(100,100,100);
color: rgb(200,200,200);
border: 1px solid rgb(100,100,100);
min-width: 15px;
}

QPushButton:hover {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(236, 100, 0, 255), stop:0.625 rgba(255, 118, 18, 255), stop:1 rgba(196, 66, 0, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:reflect, x1:0, y1:0.051, x2:0, y2:1, stop:0 rgba(209, 88, 0, 255), stop:0.625 rgba(118, 55, 8, 255), stop:1 rgba(176, 59, 0, 255));
border: 1px solid rgb(30,30,30);
}
</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>tex_type_changed(int)</slot>
 </slots>
</ui>
